demisto_sdk/commands/validate/file_validator.py

```diff
@@ -663,3 +667,11 @@ def _is_py_script_or_integration(file_path):
             return True
 
         return False
+
+    def get_content_release_identifier(self):
+        try:
+            file_content = get_remote_file('.circleci/config.yml')
```

Comment:
use the `tag` option since this file might have changed in the current branch

Reply:
tag='master' ?

Reply:
Should be branch name

Reply:
done